ci/ciseedjobcheck.yaml

```diff
@@ -0,0 +1,156 @@
+apiVersion: "v1"
+kind: "Template"
+metadata:
+  name: "container-pipeline-seedjob-check-ci-job"
+objects:
+  - apiVersion: "v1"
+    kind: "BuildConfig"
+    metadata:
+        name: ci-seed-job-check
+    spec:
+      strategy:
+        type: "JenkinsPipeline"
+        jenkinsPipelineStrategy:
+          jenkinsfile: |
+                import groovy.json.JsonSlurperClassic
+                podTemplate(
+                    cloud: 'openshift',
+                    name: 'ccp-pipeline-ci',
+                    label: 'ccp-pipeline-ci',
```

Comment:
true, my thought process was around getting these in library jenkinsfile. so kept them same. We can re-think these.

Reply:
I don't know what a library Jenkinsfile is but I remember the conversation where it was mentioned that we want to end up reusing the Jenkinsfile definitions. But in the meantime, isn't it worth keeping unique names to avoid issues like the one mentioned in earlier comment?